embedded-graphics/src/primitives/corner_radii.rs

```diff
@@ -0,0 +1,356 @@
+//! Rounded rectangle corner radii configuration
+
+use crate::geometry::Size;
+
+/// The definition of each corner radius for a rounded rectangle.
+#[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug, Default)]
+pub struct CornerRadii {
+    /// Top left corner radius
+    pub top_left: Size,
+
+    /// Top right corner radius
+    pub top_right: Size,
+
+    /// Bottom right corner radius
+    pub bottom_right: Size,
+
+    /// Bottom left corner radius
+    pub bottom_left: Size,
+}
+
+impl CornerRadii {
+    /// Create a new set of corner radii with all corners having equal values.
+    pub fn new(radius: Size) -> Self {
```

Comment:
Do we also want to demonstrate that `CornerRadii` can be constructed using its public fields as well?

Reply:
I would suggest to always use the builder in the examples. Because builders are also used in other instances, where a struct literal isn't possible, like building styles, this pattern is familiar in code that uses e-g.

Being able to create objects by using struct literals is a basic feature of rust that doesn't need to be explained in our examples IMO. And the additional methods in the builder make it more powerful than using plain struct literals.